### PR TITLE
fix video size in clip mode

### PR DESCRIPTION
**Pull request by sxian0613**
Fix Bug 12798521: Gap between the lower bound of the video and the seek bar in clip mode is too large.
The size of video in clip mode with conditions of Fullscreen or not, show bottom controller bar or not.

### CODE REVIEW

File: packages/web-components/src/player-component/player-component.ts [hunk 1]
```diff
@@ -75,6 +75,12 @@ export class PlayerComponent extends FASTElement {
         this.classList.add(this.isLive ? LiveState.ON : LiveState.OFF);
         this.classList.remove(!this.isLive ? LiveState.ON : LiveState.OFF);
 
+        if (this.isClip) {
```
Comment (giakas) [inline packages/web-components/src/player-component/player-component.ts:78]: use this.updateClass('clip-on', this.isClip) instead

File: packages/web-components/src/player-component/player-component.ts [hunk 2]
```diff
@@ -668,6 +674,12 @@ export class PlayerComponent extends FASTElement {
         }
         this.classList.add(this.isLive ? 'live-on' : 'live-off');
         this.classList.remove(!this.isLive ? 'live-on' : 'live-off');
+
+        if (this.isClip) {
```
Comment (giakas) [inline packages/web-components/src/player-component/player-component.ts:678]: same as above.

File: packages/web-components/src/player-component/player-component.ts (hunk 1 above)
Comment (giakas) [reply packages/web-components/src/player-component/player-component.ts:78]: why do you need this in constructor?

Comment (sxian0613) [reply packages/web-components/src/player-component/player-component.ts]: Yes, we can remove this line here.

File: packages/web-components/src/player-component/player-component.ts (hunk 2 above)
Comment (sxian0613) [reply packages/web-components/src/player-component/player-component.ts]: Change to use updateClass.

**Review by giakas — APPROVED** [review]

**Pull request merged**